test-dev/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+if(POLICY CMP0079)
+    cmake_policy(SET CMP0079 NEW)
+endif()
+
+if(POLICY CMP0115)
+    cmake_policy(SET CMP0115 NEW)
+endif()
+
+include_directories(
+    ${CMAKE_SOURCE_DIR}/include
+    ${CMAKE_SOURCE_DIR}/src)
+
+set(DEVTEST_SRC
+    util.c
+    main.c
+    simple_module.c
+    compare_mixer_data.c
+    ../src/md5.c
+    ../src/filetype.c
+    ../src/hio.c
+    ../src/load_helpers.c
+    ../src/period.c
+    ../src/memio.c
+    ../src/dataio.c
+    ../src/scan.c
+    ../src/loaders/itsex.c
+    ../src/loaders/sample.c
+    ../src/loaders/common.c
+    ../src/depackers/xfnmatch.c
+    ../src/win32.c
+)
+
+set(DEV_GENERATED "${CMAKE_CURRENT_BINARY_DIR}/tests-dev-generated")
+file(MAKE_DIRECTORY "${DEV_GENERATED}")
+include_directories("${DEV_GENERATED}")
+
+
+file(STRINGS "all_tests.txt" ALL_DEV_TESTS)
```

Comment:
Glad it isn't just MSVC that needs this thing now. I feel vindicated in adding it!

Reply:
It simplifies the hell of adding new unit tests into multiple build systems at once :)